vscode-trace-extension/src/extension.ts

```diff
@@ -50,11 +50,23 @@ export function activate(context: vscode.ExtensionContext): ExternalAPI {
         }
     }));
 
+    const updateUris = async (url?: string): Promise<void> => {
+        url = url ? url : getTraceServerUrl();
+        const baseUri = vscode.Uri.parse(url);
+        const extUri = await vscode.env.asExternalUri(baseUri);
+        const extUriString = extUri.toString();
+
+        tracesProvider.updateTraceServerUrl(extUriString);
+        myAnalysisProvider.updateTraceServerUrl(extUriString);
+        TraceViewerPanel.updateTraceServerUrl(extUriString);
+    };
+
     const analysisProvider = new AnalysisProvider();
     // TODO: For now, a different command opens traces from file explorer. Remove when we have a proper trace finder
     const fileOpenHandler = fileHandler(analysisProvider);
     context.subscriptions.push(vscode.commands.registerCommand('traces.openTraceFile', async file => {
-        await startTraceServerIfAvailable();
+        const url = await startTraceServerIfAvailable();
+        await updateUris(url);
```

Comment:
The `url` returned above has no trailing slash, AFAIK. As opposed to what this method call expects based on line 54. Not sure if this can be an issue.

Reply:
The code is working as expected.  I'm not sure how exactly this is handled in the code, but it is getting passed through `vscode.Uri.parse()` and my guess is that will add or remove the trailing '/' to what vscode likes.